.github/workflows/ci.yml

```diff
@@ -11,39 +11,74 @@ on:
 jobs:
   build:
     runs-on: macos-latest
+    env:
+      NOT_FORK: ${{secrets.NOT_FORK == 'true'}}
+      FORK: ${{secrets.NOT_FORK != 'true'}}
     steps:
     - uses: actions/checkout@v1
+
+    - name: Package Name
+      run: sed -i -e "s/package=\"[^\"]*\"/package=\"com.mathewsachin.fategrandautomata\"/g" AndroidManifest.xml
+      working-directory: ./FateGrandAutomata/Properties
+      if: env.NOT_FORK == 'true'
```

Comment:
I think you can leave out the == 'true', Github Actions can deal with booleans

Reply:
It doesn't. I did that before and got errors. Environment variables are always treated as strings.